### ISSUE TITLE
Pods that are not associated with any deployments

### ISSUE THREAD

**Issue by ozgurcd** [yonahd/kor#55]
Would it be useful if we implement the check to list pods that are not associated with any deployments ?  I am not sure if we consider these pods as "not used" since it is perfectly possible to create such pods, however it might also be useful if we can list them.

**Comment by yonahd** [comment]
Hi @ozgurcd
While it is rather uncommon to create a pod not from a deployment, job etc it is still something that is done. This may be something we can have people opt in on.

**Comment by ozgurcd** [comment]
So, I'll start working on this and send a PR.

**Comment by yonahd** [comment]
> So, I'll start working on this and send a PR.

I don't think this is a feature that should be worked on now. The codebase needs to be cleaned up and tests added before this can be added in a good way.
